pkgs/top-level/all-packages.nix

```diff
@@ -2613,6 +2613,10 @@ with pkgs;
 
   _86Box = callPackage ../applications/emulators/86box { };
 
+  _86Box-with-roms = callPackage ../applications/emulators/86box {
+    unfreeEnableRoms = true;
```

Comment:
If you are making this package unfree and Hydra won't build it anyway, maybe you should also `unfreeEnableDiscord`? Also, I'd use `_86Box.override`.

Reply:
Do you think it would be better to use `pkgs/by-name` instead then override?

Reply:
> Do you think it would be better to use `pkgs/by-name` instead then override?

That's unrelated to one another. You won't be able to use `pkgs/by-name` trivially for an expression that is equal to `_86box-with-roms.override { ... };`.

I'd let the automatic transition to `pkgs/by-name` do the job.